storage/path.go

```diff
@@ -27,13 +27,13 @@ func BlockDevicePath(device BlockDevice) (string, error) {
 	if device.HardwareId != "" {
 		return path.Join(diskByID, device.HardwareId), nil
 	}
-	if device.UUID != "" {
-		return path.Join(diskByUUID, device.UUID), nil
-	}
 	if len(device.DeviceLinks) > 0 {
 		// return the first device link in the list
 		return device.DeviceLinks[0], nil
 	}
+	if device.UUID != "" {
```

Comment:
could we add a comment why `DeviceLinks` takes higher priority to return here?

Reply:
It's somewhat arbitrary - there's no real reason to choose one over the other. The bug seemed to show that a UUID path was involved but because it's been impossible to reproduce, it's hard to definitively say what order is preferred. Once this gets tested on site, we can revisit if needed.